src/scaffold/hydra/config_helpers.py

```diff
@@ -1,18 +1,39 @@
 from __future__ import annotations
 
+import functools
 from abc import abstractmethod
 from copy import deepcopy
 from dataclasses import dataclass
 from functools import wraps
 from pathlib import Path
 from types import MethodType
-from typing import Callable, Dict, Optional, Type
+from typing import Any, Callable, Dict, List, Optional, Type
 
 from hydra.core.config_store import ConfigStore
+from omegaconf import DictConfig, OmegaConf
 
 CONFIG_STORE = ConfigStore.instance()
 
 
+def check_cfg_for_missing_values(allowed_missing: Optional[List[str]] = None):
+    """Ensures we have no entries with MISSING values that didnt get overwritten via yaml or cli."""
```

Comment:
I think an example usage would be nice in the docstring. After a quick not clear to me how to apply this